UserExperience_LazyLoad_Page_View.php

```diff
@@ -60,7 +60,7 @@
 		<th>Google Maps</th>
 		<td>
 			<?php Util_Ui::pro_wrap_maybe_start(); ?>
-			<p class="description w3tc-gopro-excerpt" style="padding-bottom: 10px">Lazy load google map</p>
+			<p class="description w3tc-gopro-excerpt" style="padding-bottom: 10px">( 'Lazy load google map', 'w3-total-cache' )</p>
```

Comment:
Missing PHP tags, use of translation function, and semi-colon.

Reply:
Done